### ISSUE TITLE
multiple topic sink to multiple elasticsearch index

### ISSUE THREAD

**Issue by woodsmur** [apache/pulsar#8873]
Hi,
Is there a way to send data from multiple topic to different elasticsearch indices? For example, if I have `t-2020.12.01` and `t-2020.12.02`, how could I consume these two topics and create two corresponding indices `t-2020.12.01` and `t-2020.12.02`?

**Comment by hpvd** [comment]
yes this would be helpful in many cases of bigger installations.
At this time doc is only talking about "singular" https://pulsar.apache.org/docs/en/io-elasticsearch/

Problem and possible solution looks somehow equal to https://github.com/apache/pulsar/issues/4547
